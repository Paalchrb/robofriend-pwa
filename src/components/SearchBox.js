import React from 'react';

const SearchBox = ({ searchChange }) => {
  return (
    <div>
      <input 
        aria-label='Search Robots'
        className='search-field' 
        type='search' 
        placeholder='search robots'
        onChange={searchChange}
      />
    </div>
  );
};

export default SearchBox;