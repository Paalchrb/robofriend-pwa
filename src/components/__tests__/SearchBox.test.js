import { shallow } from 'enzyme';
import SearchBox from '../SearchBox'; 
import React from 'react';


it('expect to render SearchBox component', () => {
  const wrapper = shallow(<SearchBox searchChange={() => null}/>);
  expect(wrapper).toMatchSnapshot();
});