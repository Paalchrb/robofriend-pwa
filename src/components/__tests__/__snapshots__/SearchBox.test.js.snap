// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render SearchBox component 1`] = `
<div>
  <input
    aria-label="Search Robots"
    className="search-field"
    onChange={[Function]}
    placeholder="search robots"
    type="search"
  />
</div>
`;
